instrumentation/mongodb/README.md

```diff
@@ -0,0 +1,33 @@
+# brave-instrumentation-mongodb
+
+This includes [`TraceMongoCommandListener`](src/main/java/brave/mongodb/TraceMongoCommandListener.java), a
+[`CommandListener`](https://mongodb.github.io/mongo-java-driver/3.12/driver/reference/monitoring/#command-monitoring)
+for the Mongo Java driver that will report via Brave how long each command takes, along with relevant tags like the
+collection/view name, the command's name (`insert`, `update`, `find`, etc.).
+
+This instrumentation can only be used with the synchronous MongoDB driver. Do not use it with the asynchronous or
+reactive drivers as tracing data will be incorrect.
+
+## Span properties:
+- `name`: command name (and collection/view name, if available). Examples: `find myCollection`, `listCollections`, etc.
+- `kind`: `CLIENT`
+- `remoteServiceName`: `mongodb-${databaseName}`. Example: `mongodb-myDatabase`.
+- `remoteIpAndPort`: the IP address and port number of the MongoDB server that the command was issued to
+- `error`: `Throwable` in case of failed command
+- Tags:
```

Comment:
PS do you know how mongo gets this breakdown, classifying commands by type such as write? https://docs.mongodb.com/manual/reference/command/serverStatus/#opcounters

Reply:
That classification probably comes from the wire protocol opcodes: https://docs.mongodb.com/manual/reference/mongodb-wire-protocol/#request-opcodes